test/BuildingRegistry.Tests/SnapshotVerifier/AddressPersistentLocalIdsComparerTests.cs

```diff
@@ -0,0 +1,89 @@
+﻿namespace BuildingRegistry.Tests.SnapshotVerifier
+{
+    using System.Collections.Generic;
+    using System.Linq;
+    using Be.Vlaanderen.Basisregisters.SnapshotVerifier;
+    using Building;
+    using FluentAssertions;
+    using KellermanSoftware.CompareNetObjects;
+    using KellermanSoftware.CompareNetObjects.TypeComparers;
+    using Snapshot.Verifier;
+    using Xunit;
+
+    public class AddressPersistentLocalIdsComparerTests
```

Comment:
The custom comparers don't exist anymore. I think we should rename the test classes.

Reply:
Changed it myself.